drivers/SmartThings/zigbee-humidity-sensor/src/aqara/init.lua

```diff
@@ -0,0 +1,75 @@
+local battery_defaults = require "st.zigbee.defaults.battery_defaults"
+local clusters = require "st.zigbee.zcl.clusters"
+local capabilities = require "st.capabilities"
+
+local TemperatureMeasurement = clusters.TemperatureMeasurement
+local RelativeHumidity = clusters.RelativeHumidity
+local PowerConfiguration = clusters.PowerConfiguration
+
+local FINGERPRINTS = {
+  { mfr = "LUMI", model = "lumi.sensor_ht.agl02" }
+}
+
+local configuration = {
+  {
+    cluster = TemperatureMeasurement.ID,
+    attribute = TemperatureMeasurement.attributes.MeasuredValue.ID,
+    minimum_interval = 30,
+    maximum_interval = 300,
+    data_type = TemperatureMeasurement.attributes.MeasuredValue.base_type,
+    reportable_change = 1
+  },
+  {
+    cluster = RelativeHumidity.ID,
+    attribute = RelativeHumidity.attributes.MeasuredValue.ID,
+    minimum_interval = 30,
+    maximum_interval = 3600,
+    data_type = RelativeHumidity.attributes.MeasuredValue.base_type,
+    reportable_change = 1
```

Comment:
FYI,
--- @class st.zigbee.defaults.temperatureMeasurement.TemperatureConfiguration
--- @field public cluster number TemperatureMeasurement cluster ID 0x0402
--- @field public attribute number MeasuredValue attribute ID 0x0000
--- @field public minimum_interval number 30 seconds
--- @field public maximum_interval number 300 seconds (5 mins)
--- @field public data_type st.zigbee.data_types.Int16 the Int16 class
--- @field public reportable_change number 16 (.1 C)
local measured_value_configuration = {
  cluster = zcl_clusters.TemperatureMeasurement.ID,
  attribute = zcl_clusters.TemperatureMeasurement.attributes.MeasuredValue.ID,
  minimum_interval = 30,
  maximum_interval = 300,
  data_type = zcl_clusters.TemperatureMeasurement.attributes.MeasuredValue.base_type,
  reportable_change = 16
}
--- @class st.zigbee.defaults.relativeHumidityMeasurement.HumidityMeasuredValueConfiguration
--- @field public cluster number RelatvieHumidity ID 0x0405
--- @field public attribute number MeasuredValue ID 0x0000
--- @field public minimum_interval number 30 seconds
--- @field public maximum_interval number 3600 seconds (1 hour)
--- @field public data_type st.zigbee.data_types.Uint16 the Uint16 class
--- @field public reportable_change number 100 (1%)
local relative_humidity_configuration =   {
  cluster = HumidityCluster,
  attribute = HumidityCluster.attributes.MeasuredValue,
  minimum_interval = 30,
  maximum_interval = 3600,
  data_type = HumidityCluster.attributes.MeasuredValue.base_type,
  reportable_change = 100
}
--- @class st.zigbee.defaults.battery.BatteryVoltageConfiguration
--- @field public cluster number PowerConfiguration ID 0x0001
--- @field public attribute number BatteryVoltage ID 0x0020
--- @field public minimum_interval number 30 seconds
--- @field public maximum_interval number 21600 seconds (6 hours)
--- @field public data_type st.zigbee.data_types.Uint8 the Uint8 class
--- @field public reportable_change number 1 (.1 volts)
local default_voltage_configuration = {
  cluster = zcl_clusters.PowerConfiguration.ID,
  attribute = zcl_clusters.PowerConfiguration.attributes.BatteryVoltage.ID,
  minimum_interval = 30,
  maximum_interval = 21600,
  data_type = zcl_clusters.PowerConfiguration.attributes.BatteryVoltage.base_type,
  reportable_change = 1
}

'reportable_change' also should be checked.
If you use same configuration on temperature&humitidy, you don't need to set these configuration.

Reply:
I removed same configurations